Allow passing content to Multipart::addFile.

pwhelan opened the pull request:
This way files can be added from non-file sources.

pwhelan committed:
Allow passing content to Multipart::addFile.

This way files can be added from non-file sources.

aaronpk commented on src/p3k/Multipart.php:
```diff
@@ -38,13 +38,14 @@ public function addPart($key, $value) {
     $this->_data .= 'Content-Transfer-Encoding: binary' . self::EOL;
     $this->_data .= self::EOL;
-    $this->_data .= file_get_contents($filename) . self::EOL;
+    if ($content) $this->_data .= file_get_contents($filename) . self::EOL;
```
Should this be the other way around? `if(!$content)`

pwhelan replied:
I think you're right.

pwhelan committed:
Update Multipart.php

FIX: typo that basically makes passing $content do the exact opposite of what it should.
